packages/posthog514client-rs/LICENSE

```diff
@@ -0,0 +1,20 @@
+MIT License
+Copyright (c) 2025 Tim Delisle, Nicolas Joseph
```

Comment:
That should be Fiveonefour Labs Inc now

Reply:
Copied this from the moose LICENSE.  Should I update both?

Reply:
yeah good call